R/colors_race.R

```diff
@@ -8,21 +8,31 @@
 #'
 #' @noRd
 make_race_palette <- function(white, black, hispanic, asian, other) {
+    if (missing(white) & missing(black) & missing(hispanic) & missing(asian) & missing(other)) {
+      stop("You must specify at least one race category", call. = FALSE)
```

Comment:
So, I think that this actually needs to be handled a bit differently - the function was originally developed so that, in the absence of any arguments, it would default to looking for "white" and "asian" and "hispanic" and so on. I believe how you've constructed it here requires people to put it in, even if the values are appropriately named for the default. Could you update the code so that it maintains the default behavior?

Reply:
Makes sense, code has been updated to maintain the default behavior!